content/correspondence/reference/API endpoints/initialize/_index.no.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Initialize
+linktitle: Initialize
+description: Endpoint for initializing a correspondence.
+
+weight: 60
+toc: true
+---
+
+## Endpoint
+
+POST /correspondence/api/v1/correspondence
+
+## Authentication
+
+Use the endpoints in the authentication folder to authenticate before initializing.
+
+## Response
+
+### Response codes
+- 200 OK: The correspondence have been succesfully initialized
+
+  Refer to problem details in response body for further information.
+- 401 Unauthorized: Indicates a missing, invalid or expired authorization header.
+- 403 Forbidden: Indicates that required scope or Platform Access Token is missing or invalid.
+
+### Content-Type
+- application/json
+
+### Response body 
+The response body is formatted as an 
+[InitializeCorrespondencesExt](https://github.com/Altinn/altinn-correspondence/blob/main/src/Altinn.Correspondence.API/Models/InitializeCorrespondencesExt.cs)
+and serialized as a JSON string.
+
+
+### Response body properties
```

Comment:
The response body is the [object](https://github.com/Altinn/altinn-correspondence/blob/main/src/Altinn.Correspondence.API/Models/InitializeCorrespondencesResponseExt.cs) `InitializeCorrespondenceResponseExt`. This documentation below does not reflect the fields in the object.